Cleanup and optimizations and support for #87

KingArthurLV opened the pull request:
Since I am a bit effiecency freak, I went trough some xml files and made some cosmetic changes for some standartization and unity.
It is good practice to use tab indentions instead of space. It also saves some file size space.
Removed also redundant zeros in support offsets and xyz to keep it tidy and efficient.

<img width="1442" alt="Screenshot 2023-05-26 102919" src="https://github.com/ddewar/FS22_UniversalAutoloadModhubAddon/assets/130602332/6d4eefdb-d5b7-4194-9a7b-a0be9c8c1347">

ddewar commented:
So unfortunately, this rendered out to a giant diff. Can you tell me what all you've changed with this? Also, it looks like a few vehicles were added support as well? Do you have links to the modhub items for what was added?

KingArthurLV commented:
Sorry for late replay, had busy work days.
So about giant diff. As I commented in commits, main changes are not functional, but code formating. Replaced indents from space to tabs, as this is more professional and also saves file size space in long run. It is better to hit 2 or 3 tabs instead of multiple space. Also I removed all unnecesary "0" in all offset and xyz configurations, to keep code unified and clean.

At the end I also added support for 2 mods:
- [FS22_HomemadeFlatbedTrailer](https://www.farming-simulator.com/mod.php?lang=en&country=us&mod_id=267779&title=fs2022)
- [FS22_OT2000](https://www.farming-simulator.com/mod.php?lang=en&country=us&mod_id=268840&title=fs2022)

ddewar commented:
Perfect. Let me verify those two mods and go from there. Thanks.

ddewar commented:
Looking pretty good. If I was going to be picky, I would say the FS22_OT2000 mod has some slight clipping on the front side of the zone causing some bouncing of the loaded objects sometimes, but I'm not sure it causes enough issue to slow this down. We can always come back and refine it if there's concern. Also worth noting, the bale height is at a height where none of the bale types actually can use the full height, but that doesn't make a difference either. Looks good overall. Will get this approved unless you really want to address either of those issues.

KingArthurLV commented:
Nice, thank You!
I agree with You, we can come back later. I am working on some kind of standart, so I will be able to go trough all supports and recheck and make neccesarry minor adjustments.
